### DISCUSSION TITLE
Heroku loaddata error

### DISCUSSION THREAD

**Discussion by hainhtat** [WeblateOrg/weblate#5159] [Q&A] [answered]
Hello, I am trying to deploy my django project on heroku and i got the deserialization error, permission matching query does not exist while I run the command 'heroku run python manage.py loaddata '. Please help me. Thank you. Here is my github repo for project. https://github.com/hainhtat/carzone-github. Here is the screenshot of the error.
<img width="682" alt="Screen Shot 2021-01-08 at 12 14 23 AM" src="https://user-images.githubusercontent.com/47028268/103926073-d2bb8f80-5146-11eb-8b8b-575f097c10a0.png">

**Comment by nijel** [comment, marked as answer]
It seems that the auth.user objects are not in sync with real permissions. Not sure how it's possible, but it has no practical impact as Weblate no longer uses Django permissions.

Quick fix: drop all auth.user objects from the json file. They are no longer used by Weblate, but the model cannot be removed from the database because of Django limitations.
